new/LIT6228MGBebaGeorge.xml

```diff
@@ -0,0 +1,112 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6228MGBebaGeorge" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: George, son of the bishop in Bǝbā, is
+                    being revived</title>
+                <title xml:id="t2" xml:lang="en">Resurrection of George, son of the bishop who was
+                    minister in the chapel of Bǝbā</title>
```

Comment:
a/the minister?

Reply:
I copied the original title proposed by Strelcyn. It reads like above. Shall I  add "the"?

Reply:
Up to you, I think it reads a little better with article, but it's not dramatic :)